src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -110,7 +110,7 @@ public function __toString()
         }
 
         if (null !== $this->samesite) {
-            $str .= '; samesite='.$this->samesite;
+            $cookie .= '; samesite='.$this->samesite;
```

Comment:
Nice catch. Could you update https://github.com/symfony/symfony/blob/master/src/Symfony/Component/BrowserKit/Tests/CookieTest.php#L19 to cover this?

Reply:
Thanks, I just pushed it